src/core/reducers/errors.js

```diff
@@ -17,15 +17,19 @@ function getMessagesFromError(error) {
 
     Object.keys(error.response.data).forEach((key) => {
       const val = error.response.data[key];
-      val.forEach((msg) => {
-        if (key === 'non_field_errors') {
-          // Add generic messages for the API response.
-          apiMessages.push(msg);
-        } else {
-          // Add field specific error messages.
-          apiMessages.push(`${key}: ${msg}`);
-        }
-      });
+      if (Array.isArray(val)) {
+        val.forEach((msg) => {
+          if (key === 'non_field_errors') {
```

Comment:
Constants introduce a level of abstraction so I think they are only good if more than one function uses the string. I can add a comment linking to the API docs http://addons-server.readthedocs.io/en/latest/topics/api/overview.html#bad-requests